bcachefs/utils.h

```diff
@@ -0,0 +1,29 @@
+/* Include Guard */
+#ifndef INCLUDE_UTILS_H
+#define INCLUDE_UTILS_H
+
+/**
+ * Includes
+ */
+
+#include <stdint.h>
+
+/* Extern "C" Guard */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+uint64_t benz_get_flag_bits(const uint64_t bitfield, uint8_t first_bit, uint8_t last_bit);
```

Comment:
Resolves [this conversation](https://github.com/mila-iqia/bcachefs/pull/25#discussion_r744191236)